thoth/metrics_exporter/jobs/inspection.py

```diff
@@ -19,7 +19,6 @@
 
 import logging
 
-from thoth.storages import GraphDatabase
```

Comment:
Also `get_inspection_results_per_identifier` is a class method

Reply:
It's a static method in sources - it does not need graph database instance. Or am I missing something?

Reply:
All good thanks :)